webapp/src/components/backstage/playbook_edit.tsx

```diff
@@ -621,7 +629,7 @@ const PlaybookEdit: FC<Props> = (props: Props) => {
                 title={'Confirm discard'}
                 message={'Are you sure you want to discard your changes?'}
                 confirmButtonText={'Discard Changes'}
-                onConfirm={props.onClose}
+                onConfirm={() => onClose()}
```

Comment:
One of my classic nits (feel free to ignore, this is just me being super picky):
```suggestion
                onConfirm={onClose}
```

Reply:
Almost, but that won't work here; `onConfirm` tries to pass in a `checked: boolean` which conflicts with the onClose signature. :)

Reply:
Oh no :( I don't understand why it lets you pass a function with no arguments, then.  the type safety